source/com/athaydes/specks/assertion/assertions.ceylon

```diff
@@ -43,16 +43,23 @@ shared AssertionResult expect<Element>(Element actual, Matcher<Element> matcher)
 shared AssertionResult expectCondition(Boolean expectedToBeTrue)
         => expect(expectedToBeTrue, toBe(identicalTo(true)));
 
-"Platform independent name of an Exception"
-shared String platformIndependentName(Type<Exception>|Throwable exception) =>
+"Platform independent name of a Throwable"
+shared String platformIndependentName(Type<Throwable>|Throwable exception) =>
         exception.string.replace("::", ".");
 
+"A value to signal \"no comparison should be done\" in an assertion"
```

Comment:
That's not how it's being used, it seems you just pass this value in to signal `expectToThrow`'s message should be not used??